airbyte-integrations/bases/base-normalization/normalization/transform_catalog/destination_name_transformer.py

```diff
@@ -150,6 +150,11 @@ def __normalize_non_column_identifier_name(
         if truncate:
             result = self.truncate_identifier_name(input_name=result, conflict=conflict, conflict_level=conflict_level)
         result = self.__normalize_identifier_case(result, is_quoted=False)
+        if result[0].isdigit():
+            if self.destination_type == DestinationType.MSSQL:
+                result = "_" + result
+            elif self.destination_type == DestinationType.ORACLE:
+                result = "ab_" + result
```

Comment:
@ChristopheDuong I need to made this to work with Oracle and MSSQL. Oracle doesn't allow tables to start with `_` so I added the prefix `ab`. What do you think?